pkg/org/org.go

```diff
@@ -47,43 +42,9 @@ func (e *AppError) String() string {
 }
 
 const (
-	ENVIRONMENT_URI      string = "/environments"
 	CONNECTED_AGENTS_URI string = "/identity/connected-agents"
 )
 
-func GetEnvironments(ArmoryCloudAddr *url.URL, accessToken *string) ([]Environment, error) {
-	environmentUrl := &url.URL{
-		Scheme: ArmoryCloudAddr.Scheme,
-		Host:   ArmoryCloudAddr.Host,
-		Path:   ENVIRONMENT_URI,
-	}
-	request := util.NewHttpRequest("GET", environmentUrl.String(), nil, accessToken)
-	request.BearerToken = accessToken
-	resp, err := request.Execute()
-	if err != nil {
-		return nil, errors.New("unable to retrieve environments to login to; please try again")
-	}
-	defer resp.Body.Close()
-	dec := json.NewDecoder(resp.Body)
-	if resp.StatusCode == 200 {
-		var environments []Environment
-		err = dec.Decode(&environments)
-		if err != nil {
-			return nil, err
-		}
-		return environments, nil
-	}
-
-	var errorResponse *ApiError
-	err = dec.Decode(&errorResponse)
-	if err != nil {
-		return nil, err
-	}
-
-	errContext := fmt.Sprintf(". ErrorId: %s, Desc: %v", errorResponse.ErrorId, errorResponse.Errors)
-	return nil, errorUtils.NewErrorWithDynamicContext(ErrEnvironmentRetrieval, errContext)
-}
-
```

Comment:
Deleted the environments function as its no longer used. Before this PR it was used by `cmd/login.go`, but I've refactored that code to use the new cloud client in configuration package.